src/main/java/com/igot/cb/cache/AccessSettingRuleCacheMgr.java

```diff
@@ -91,15 +95,16 @@ private void loadAccessSettingRules() {
                         null, null);
                 cachedAccessSettingRules = accessSettingRuleMapList.stream()
                         .map(record -> new CachedAccessSettingRule(
-                                (String) record.get("contextId"),
-                                (String) record.get("contextIdType"),
+                                (String) record.get("contextid"),
+                                (String) record.get("contextidtype"),
```

Comment:
why this change required?
does the redis returns all small case ?? or we haven't added these in cassandratablecolumn.properties ??